test/DocumentFormat.OpenXml.Tests/OpenXmlDomTest/DocumentOpenTests.cs

```diff
@@ -60,7 +60,7 @@ public void RewriteMalformedUri()
                         Count++;
                         Assert.Equal(Id, id);
                         Assert.Equal("/xl/worksheets/_rels/sheet1.xml.rels", partUri.OriginalString);
-                        Assert.Equal("mailto:one@", uri);
+                        Assert.Equal("mailto:test@test.com;%20test2@test.com;%252test3@test.com;%20test3@test.com;%20test4@test.com;%20test5@test.com?subject=Unsubscribe%20Request&body=Please%20unsubscribe%20me%20from%20all%20future%20communications", uri);
```

Comment:
Can you keep the original test and add a new one? including the updated malformed_uri.xlsx in addition to the original one

Reply:
Yep yep, let me move that out.